src/colour.c

```diff
@@ -31,7 +31,9 @@ const glColour cGrey80     = { .r=0.80, .g=0.80, .b=0.80, .a=1. }; /**< Grey 80%
 const glColour cGrey70     = { .r=0.70, .g=0.70, .b=0.70, .a=1. }; /**< Grey 70% */
 const glColour cGrey60     = { .r=0.60, .g=0.60, .b=0.60, .a=1. }; /**< Grey 60% */
 const glColour cGrey50     = { .r=0.50, .g=0.50, .b=0.50, .a=1. }; /**< Grey 50% */
+const glColour cGrey45     = { .r=0.45, .g=0.45, .b=0.45, .a=1. }; /**< Grey 45% */
```

Comment:
I'm not a big fan of adding so many colors. In fact, I think we should reduce them a lot more. Are these new colors absolutely necessary?

Reply:
Yeah, there's a pretty big difference between a 5% gradation.